android/src/main/java/cz/ackee/extensions/android/ContextExtensions.kt

```diff
@@ -127,6 +131,16 @@ fun Context.isDarkModeOn(): Boolean {
     return currentNightMode == Configuration.UI_MODE_NIGHT_YES
 }
 
+/**
+ * Open Play Store with the application ID from provided context.
+ */
+fun Context.openStore() {
+    try {
+        startActivity(Intent(Intent.ACTION_VIEW, Uri.parse("market://details?id=$packageName")))
+    } catch (exception: ActivityNotFoundException) {
```

Comment:
i would prefer to query activities and check if any can handle the intent rather than try/catching.

Reply:
fixed